topics/introduction/tutorials/galaxy-intro-strands/tutorial.md

```diff
@@ -124,7 +124,7 @@ There are [many ways to get data into a Galaxy instance]({% link topics/galaxy-i
 >
 > 1. *Click* on the **Get Data** toolbox to expand it.
 >
->    ![The Get Data toolbox](../../images/101_01.png)
+>    ![The Get Data toolbox](../../images/getdata.PNG)
```

Comment:
Could you please rename your file to overwrite the old image? Currently there are two images intending to describe the same thing, and that first image might be used elsewhere in the GTN. If you replace the old image with your new one, then everywhere that image is used will be automatically updated.

Reply:
I did not remove the first image from the image folder, I replaced a particular image using the first image as a guide but did not fit into the content description. i will go ahead to revert it back to normal.

Reply:
Hi @sandygudie I'm sorry for the confusion. 

I was thinking to replace the old image since it is used in multiple places throughout the GTN:

```
topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md
178:>     ![UCSC Main tool in tools section](../../images/101_01.png)

topics/introduction/tutorials/galaxy-intro-101/tutorial.md
150:> >     ![`UCSC Main table browser` menu entry](../../images/101_01.png)

topics/introduction/tutorials/galaxy-intro-strands/tutorial.md
127:>    ![The Get Data toolbox](../../images/101_01.png)
139:>    ![Click on UCSC Main table browser](../../images/101_01.png)
```

if we replace the 101_01.png with your new image, then we fix all of those tutorials. However many of them may be related to "UCSC Main" specifically which doesn't appear in your image (that's fine!) So I guess we keep it as a separate image then. I'll make the relevant changes but overall this is fine now.

Reply:
okay